types & grammar/ch1.md

```diff
@@ -17,21 +17,21 @@ Em outras palavras, se tanto o motor (engine) quanto o desenvolvedor tratam o va
 
 Isso não é uma definição perfeita. Mas é bom o suficiente para essa discussão. E é consistente com a maneira que JS descreve-se.
 
-# A Type By Any Other Name...
+# Um tipo por qualquer outro nome...
```

Comment:
Thiago, a melhor definição que achei de *"by any other name"*, seria como dizer **o que importa é o que ele é, não pelo o que é chamado"*.

*What matters is what something is, not what it is called.*

Pensei em traduzir dessa forma. **"O que importa em um tipo é o que ele é, não pelo quê é chamado"**. O que acha?